conf/evolutions/default/4.sql

```diff
@@ -0,0 +1,14 @@
+# --- Cascade deletes to make dropping data easier (1 step instead of 3)
+# --- !Ups
+
+ALTER TABLE yarn_app_heuristic_result DROP FOREIGN KEY yarn_app_heuristic_result_f1;
```

Comment:
@paulbramsen , this requires evolutions to be enabled in production right? I think people usually disable play evolutions in production due to the risk of data being erased during deployment with schema changes.